src/order/boolean_algebra.lean

```diff
@@ -574,9 +579,16 @@ postfix `ᶜ`:(max+1) := compl
 
 /-- This class contains the core axioms of a Boolean algebra. The `boolean_algebra` class extends
 both this class and `generalized_boolean_algebra`, see Note [forgetful inheritance]. -/
-class boolean_algebra.core (α : Type u) extends bounded_distrib_lattice α, has_compl α :=
+class boolean_algebra.core (α : Type u) extends distrib_lattice α, has_compl α,
+  has_top α, has_bot α :=
```

Comment:
Does extending bounded_order not work here? A brief comment about why would be good